tools/depends/target/libspdlog/Makefile

```diff
@@ -0,0 +1,67 @@
+-include ../../Makefile.include
+DEPS = Makefile
+
+# lib name, version
+LIBNAME=spdlog
+VERSION=1.5.0
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.gz
+
+CMAKE_OPTIONS= \
+	-DSPDLOG_BUILD_EXAMPLE=OFF \
+	-DSPDLOG_BUILD_TESTS=OFF \
+	-DSPDLOG_BUILD_BENCH=OFF \
+	-DSPDLOG_INSTALL=ON \
+	-DSPDLOG_FMT_EXTERNAL=ON
+
+ifeq ($(CROSS_COMPILING), yes)
+  DEPS += ../../Makefile.include
+else
+  CXXFLAGS += -std=c++11
+  ROOT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+
+  ifeq ($(PLATFORM),)
+    PLATFORM = native
+    TARBALLS_LOCATION = $(ROOT_DIR)
+    BASE_URL := http://mirrors.kodi.tv/build-deps/sources
+    RETRIEVE_TOOL := curl
+    RETRIEVE_TOOL_FLAGS := -Ls --create-dirs -f -O
+    ARCHIVE_TOOL := tar
+    ARCHIVE_TOOL_FLAGS := --strip-components=1 -xf
+    CMAKE := cmake -DCMAKE_INSTALL_PREFIX=$(PREFIX)
+  endif
+endif
+
+LIBDYLIB=$(PLATFORM)/build/libfmt.a
+
+.PHONY: .installed-native
+
+all: .installed-$(PLATFORM)
+
+$(TARBALLS_LOCATION)/$(ARCHIVE):
+	cd $(TARBALLS_LOCATION); $(RETRIEVE_TOOL) $(RETRIEVE_TOOL_FLAGS) $(BASE_URL)/$(ARCHIVE)
+
+$(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
+ifeq ($(PREFIX),)
+	@echo
+	@echo "ERROR: please set PREFIX to the kodi install path e.g. make PREFIX=/usr/local"
+	@exit 1
+endif
+	rm -rf $(PLATFORM); mkdir -p $(PLATFORM)
+	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
+	cd $(PLATFORM); rm -rf build; mkdir -p build
+	cd $(PLATFORM)/build; $(CMAKE) $(CMAKE_OPTIONS) ..
```

Comment:
```Makefile
	cd $(PLATFORM); $(CMAKE) -B build $(CMAKE_OPTIONS)
```

Reply:
This is how it's done for lots of other dependencies. A quick `grep` reveals
* libandroidjni
* libcec
* libfmt (where I copied this from)
* mariadb
* p8-platform
* rapidjson
* taglib

Only `libjpeg-turbo` uses your approach.

Reply:
this is really a minor. As Montellese points out, we historically use it as is and both do the same thing, so it's a matter of taste.